src/navigation/pathnavigator.cpp

```diff
@@ -462,6 +465,33 @@ scripting::LuaLibrary PathNavigator::luaLibrary() {
                 "that roll must be included for the target up direction to be taken "
                 "into account."
             },
+            {
+                "zoomToFocus",
+                &luascriptfunctions::zoomToFocus,
+                "[duration]",
+                "Zoom linearly to the current focus node, using the default distance."
+                "The optional input parameter specifies the duration for the motion."
```

Comment:
Which unit is the duration in?  Assuming seconds, of course

Reply:
Yep, seconds